src/bin/cmd/client.rs

```diff
@@ -149,6 +159,21 @@ impl HTTPNodeClient {
 		e.reset().unwrap();
 	}
 
+	pub fn validate_chain(&self, fast_validation: bool) {
```

Comment:
how come this is a new method, rather than a changed method?

Reply:
I don't really get what you mean here by a changed method?

Reply:
I mean that it looks like you're adding a validate_chain method to HTTPNodeClient,
whereas elsewhere in this commit, there are changes to existing validate_chain methods.

Reply:
ah ok, will call it, `verify_chain` to make it clear it's a new method.

Reply:
I don't object to the name. If it behaves identical to those others, then the same name is appropriate. But I just wondered, since I'm not very familiar with this code, why HTTPNodeClient didn't need such a method before and does now.